lib/Extension/ClassMover/Application/ClassCopy.php

```diff
@@ -72,7 +72,7 @@ public function copyFile(ClassCopyLogger $logger, string $srcPath, string $destP
             // if the src is not the same as the globbed src, then it is a wildcard
             // and we want to append the filename to the destination
             if ($srcPath !== $globPath) {
-                $globDest = Path::join($destPath, Path::getFilename($globPath));
+                $globDest = Path::join($destPath, basename($globPath));
```

Comment:
hm, which this change?

Reply:
ok, old version did basically the same thing.

Reply:
Yeah, I basically replaced the function by it's implementation.